### ISSUE TITLE
PriviledgedOperation or PriviledgedOperations?

### ISSUE THREAD

**Issue by portaldotjay** [microsoftgraph/microsoft-graph-docs#8243]
Doc says 'DeviceManagementManagedDevices.PrivilegedOperation.All' (No 's' at the end of operations). Permission doesn't exist but 'DeviceManagementManagedDevices.PrivilegedOperations.All' does but still get 403 when trying. 

---
#### Document Details

⚠ *Do not edit this section. It is required for docs.microsoft.com ➟ GitHub issue linking.*

* ID: ad182eee-d0d2-801f-4b4d-9bb7abe87f9d
* Version Independent ID: a706cc6c-e1b6-a340-17c3-e9abc3f5295d
* Content: [setDeviceName action - Microsoft Graph beta](https://docs.microsoft.com/en-us/graph/api/intune-devices-manageddevice-setdevicename?view=graph-rest-beta&viewFallbackFrom=graph-rest-1.0)
* Content Source: [api-reference/beta/api/intune-devices-manageddevice-setdevicename.md](https://github.com/microsoftgraph/microsoft-graph-docs/blob/master/api-reference/beta/api/intune-devices-manageddevice-setdevicename.md)
* Product: **intune**
* Technology: **microsoft-graph**
* GitHub Login: @dougeby
* Microsoft Alias: **MSGraphDocsVteam**

**Comment by triplanedave** [comment]
Thanks for the feedback.  We'll have a dev take a look at this.  In Graph Explorer, it says the permission required is without the 's'.  I get an error trying to POST an update to my device (either Windows or Android) as a global admin, so we'll take a look.

**Comment by portaldotjay** [comment]
I was digging into this a little more. Outside of Graph Explorer, I've been able to assign DeviceManagementManagedDevices.PrivilegedOperations.All and perform setDeviceName using PowerShell without issues. This got me thinking a little...

I was able to capture some of what's going on using Postman Interceptor. When consent is clicked, it's requesting DeviceManagementManagedDevices.PrivilegedOperation.All permissions and an error is generated:

The+application+'Graph+explorer+(official+site)'+asked+for+scope+'DeviceManagementManagedDevices.PriviligedOperation.All'+that+doesn't+exist+on+the+resource+'00000003-0000-0000-c000-000000000000'

In Graph Explorer, are the permissions generated by the Docs data? If so, maybe it's just the doc that needs changed.
